.backportrc.json

```diff
@@ -1,13 +1,14 @@
 {
   "targetBranchChoices": [
     { "name": "main", "checked": true },
+    "8.10",
     "8.9",
     "8.8",
```

Comment:
As there won't be any minor release for 8.8, I think we can remove this line. Be careful of the trailing comma as well.

Reply:
Done!